testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/BaseTestAvroSchemaEvolution.java

```diff
@@ -0,0 +1,323 @@
+        assertThat(query(selectStarStatement))
+                .containsExactlyInOrder(
+                        prepareQueryResultRows(
+                                new Object[][] {
+                                        {"stringA0", 0}
+                                }));
+
+        alterTableSchemaTo(ADDED_COLUMN_SCHEMA);
+        assertThat(query(columnsInTableStatement))
+                .containsExactlyInOrder(
+                        prepareShowColumnsResultRows(new String[][] {
+                                {"string_col", "varchar", "", ""},
+                                {"int_col", "integer", "", ""},
+                                {"int_col_added", "integer", "", ""}
+                        }));
+        assertThat(query(selectStarStatement))
+                .containsExactlyInOrder(
+                        prepareQueryResultRows(
+                                new Object[][] {
+                                        {"stringA0", 0, 100}
+                                }));
+
+        alterTableSchemaTo(REMOVED_COLUMN_SCHEMA);
+        assertThat(query(columnsInTableStatement))
+                .containsExactlyInOrder(
+                        prepareShowColumnsResultRows(new String[][] {
+                                {"int_col", "integer", "", ""}
+                        }));
+        assertThat(query(selectStarStatement))
+                .containsExactlyInOrder(
+                        prepareQueryResultRows(
+                                new Object[][] {
+                                        {0}
+                                }));
+
+        alterTableSchemaTo(RENAMED_COLUMN_SCHEMA);
+        assertThat(query(columnsInTableStatement))
+                .containsExactlyInOrder(
+                        prepareShowColumnsResultRows(new String[][] {
+                                {"string_col", "varchar", "", ""},
+                                {"int_col_renamed", "integer", "", ""},
+                        }));
+        assertThat(query(selectStarStatement))
+                .containsExactlyInOrder(
+                        prepareQueryResultRows(
+                                new Object[][] {
+                                        {"stringA0", null}
+                                }));
+    }
+
+    @Test(groups = AVRO)
+    public void testSchemaWhenUrlIsUnset()
+    {
+        assertThat(query(columnsInTableStatement))
+                .containsExactlyInOrder(
+                        prepareShowColumnsResultRows(new String[][] {
+                                {"string_col", "varchar", "", ""},
+                                {"int_col", "integer", "", ""}
+                        }));
+        assertThat(query(selectStarStatement))
+                .containsExactlyInOrder(
+                        prepareQueryResultRows(
+                                new Object[][] {
+                                        {"stringA0", 0}
+                                }));
```

Comment:
We can just select rows and `SHOW COLUMN` might not be required.

Reply:
This was how original TestAvroSchemaEvolution test worked. I don't wan't to change that logic in this PR.